client.js

```diff
@@ -159,6 +160,11 @@ bot.on('message', function (message) {
                 message.author.send('Sorry, I didn\'t recognized that command.  Please check your syntax and try again. ' + helpMsg);
                 break;
         }
+        // message the general channel that commands are not recognized in this channel if command detected:
+        if (!message.author.bot && message.channel.type === 'text' && message.channel.id !== COMMAND_CHANNEL){
+            bot.channels.cache.get(COMMAND_CHANNEL).send(`Hi <@${message.author.id}>, I received your command but deleted it from the \`general\` channel to keep things tidy.  In the future, please use this channel instead or send me a direct message.  Thanks!`)
```

Comment:
Do we want to spam the command channel with this? Or should we just PM the user directly? I also don't necessarily like the idea of *publically* shaming users for making a mistake. Deleting the message has the advantage of hiding the mistake quickly, but then we announce about it loudly... :/

Reply:
I liked how it works when you are @'ed from the channel you are suppose to use.  If you think there's a shame factor though we can make it a PM.

Reply:
Yeah, I suppose it's fine. It's more like an FYI, I suppose. There's technically other cool stuff we could do, like "click the checkmark reaction to acknowledge this message" and then delete that too... but lots of extra work for no real reason :D